multi_host_migration/tests/migration_multi_host_with_stress.py

```diff
@@ -42,37 +42,53 @@ def __init__(self, test, params, env):
             self.dsthost = self.params.get("hosts")[1]
             self.is_src = params["hostid"] == self.srchost
             self.vms = params["vms"].split()
-            self.login_timeout = int(params.get("login_timeout", 360))
             self.need_cleanup = params.get("need_cleanup", "yes") == "yes"
+            self.driver_load_cmd = params.get("driver_load_cmd")
+            if self.driver_load_cmd:
+                self.nic_index = 1
+                self.login_timeout = 120
+            else:
+                self.nic_index = 0
+                self.login_timeout = int(params.get("login_timeout", 240))
+            self.bg = None
 
         def migration_scenario(self):
             def clean_up(vm):
                 kill_bg_stress_cmd = params.get("kill_bg_stress_cmd",
                                                 "killall -9 stress")
 
                 logging.info("Kill the background test in guest.")
-                session = vm.wait_for_login(timeout=self.login_timeout)
-                s = session.cmd_status(kill_bg_stress_cmd)
-                if s:
-                    raise error.TestFail("Failed to kill the background"
-                                         " test in guest.")
+                session = vm.wait_for_login(timeout=self.login_timeout,
+                                            nic_index=self.nic_index)
+                if self.params.get("bg_stress_test") == "driver_load":
+                    if self.bg and self.bg.is_alive():
+                        self.bg.join()
+                    output = session.cmd_output("ipconfig || ifconfig")
+                    logging.info("Guest network status:\n %s" % output)
+                    session.cmd(self.driver_load_cmd)
```

Comment:
Do you mean to load or unload driver here? Since it looks you tend to 'clean up' here.